operations/firmware/Firmware Upgrade using SPP on HPE ProLiant Servers.md

```diff
@@ -0,0 +1,105 @@
+# Firmware Upgrade using SPP on HPE ProLiant Servers
+#### Link to download SPP Image : 
+https://techlibrary.hpe.com/us/en/enterprise/servers/products/service_pack/spp/index.aspx
+
+* Naming convention for example : 2023.04.00.00 that's the ISO image released in 4th Month of 2023
+
+* Download the latest version or the version you needed.
+
+* We can either deploy individual firmware component or deploy SPP ISO image as a whole
+
+### How to deploy individual firmware component?
+
+####  Get the individual firmware component from SPP ISO image:
+
+* If we mount the SPP ISO image or extract the iso image, we can see there is “contents.html” file present.
+* For example, if we want to flash the BIOS/ROM for DL360 Gen11 server then find for DL360 in “Description” column, and you see the relevant component to flash through ILO. Note the extension should be fwpkg if we are flashing through ILO.
+* Get the location of the file/package where it is located in the extracted/mounted ISO image. Usually, it will be present in the folder “Packages”
+
+#### Steps to deploy individual components:
+
+* Login to the ILO page in web browser 
+* Go to the Firmware&OS Software at the left section
+* Select upload to iLO Repository
+* Select the file where it is located and click on upload.
+* Go to iLO Repository, you should be able to see the uploaded component. Click on Install Component and then Click on TPM Override then click on add to the Installation queue
+* You can check for the status message in Installation Queue. If you see any failed instance in Queue, then delete that instance else installation of next component would not get start if expiration is set to never for the failed instance.
+
+#### How to deploy SPP ISO image as a whole
+
+**Pre-requisites:**

```

Comment:
Does node need to be powered off at this point?

Reply:
Correct, we get F9 option after rebooting the node. I will add it